client/container.go

```diff
@@ -106,3 +106,14 @@ func (client *APIClient) ContainerStartExec(execid string, config *types.ExecSta
 
 	return client.hijack("/exec/"+execid+"/start", url.Values{}, config, header)
 }
+
+//ContainerRename rename a container.
```

Comment:
// ContainerRename renames a container.

add a whitespace after // and use renames please.